.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    "no-proto": 0,
+    quotes: "off",
```

Comment:
The `quotes` rule is set to "off", which means that ESLint will not enforce any specific quote style. This might lead to inconsistent quote usage in your code. Consider setting it to "single" or "double" to enforce consistency, unless the extended configuration already handles this.